### ISSUE TITLE
Train Initial Demand Forecast Model

### ISSUE THREAD

**Issue by pbtrad** [pbtrad/balancing_market#1]
Develop a basic demand forecasting model using TensorFlow. The model will use historical energy demand data from EirGrid to predict future demand patterns.

Todo:

Load processed training data from ml_models/data/processed/training_data.csv.
Build TensorFlow Sequential model with dense layers.
Train the model using MSE loss and Adam optimizer.
Save the trained model as ml_models/models/model.h5.
Evaluate performance on validation/test data.
